common.go

```diff
@@ -36,6 +36,9 @@ func getCopyOptions(store storage.Store, reportWriter io.Writer, sourceSystemCon
 	destinationCtx := getSystemContext(store, nil, "")
 	if destinationSystemContext != nil {
 		*destinationCtx = *destinationSystemContext
+		if destinationCtx.OSChoice == "windows" {
+			sourceCtx.OSChoice = "windows"
+		}
```

Comment:
This is almost certainly the wrong location for this logic.

Where is the _actual_ disconnect between users’ intent and in-process data structures coming from?